controllers/apply/form-router-next/lib/form-model.js

```diff
@@ -128,10 +156,23 @@ class FormModel {
             stripUnknown: true
         });
 
+        function matchesFeaturedCriteria(item, message) {
+            if (item.includeBaseError) {
+                return item.param === message.param;
+            } else {
+                return item.param === message.param && message.type !== 'base';
+            }
+        }
+
         const normalisedErrors = normaliseErrors({
             validationError: error,
             errorMessages: this.messages,
             formFields: this.allFields
+        }).map(message => {
+            message.isFeatured = this.featuredErrorsAllowList.some(item =>
```

Comment:
Adds a `isFeatured` flag to error messages to mark that the error is in the allow list.